app/views/users/reactivate_profile/index.html.slim

```diff
@@ -4,7 +4,9 @@ h1.h3.my0 = t('forms.reactivate_profile.title')
 p.mt-tiny.mb0 = t('forms.reactivate_profile.instructions')
 = simple_form_for(@reactivate_profile_form, url: reactivate_profile_path,
   html: { autocomplete: 'off', method: :post, role: 'form' }) do |f|
+  - field_name = "#{f.object_name}[#{:recovery_code}][]"
```

Comment:
the `#{:recovery_code}` is a little redudant...it can just be `recovery_code`, like:

```ruby
field_name = "#{f.object_name}[recovery_code][]"
```